release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -422,6 +428,21 @@ In the table, features are marked with the following statuses:
 AMD and Intel 64-bit architectures (x86-64-v2) will still be supported.
 ====
 
+[id="ocp-4-10-docker-config-location-deprecation"]
+==== Default Docker configuration location Deprecated
+Previously, `oc` commands that used a registry configuration obtained credentials from the Docker configuration location, which was `~/.docker/config.json` by default. This has been deprecated and will be replaced by a podman configuration location in a future version of {product-title}.
+
+[id="ocp-4-10-empty-file-support-deprecation"]
+==== Empty file support deprecation
+
+Support for empty files using the `--registry-config` and `--to` flags in `oc registry login` has been deprecated. It will be removed in a future version of {product-title}.
+
+[id="ocp-4-10-writing-to-stdout-option-deprecation"]
+==== Return support for `-` and `stdout` deprecation
+
+Return support for `-` and `stdout` in the `oc registry login` command been deprecated. It will be removed in a future version of {product-title}.
```

Comment:
Maybe we should mention the context? 

- Which is the same as above:  the `--registry-config` and `--to` flags 
- does it make sense to squash these two sections together?